docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.4'
+
+services:
+  dev:
+    image: zooniverse-text-editor:latest
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    entrypoint:
+      - "yarn"
+    command: ["start"]
+    ports:
+      - "3000:3000"
+    volumes:
+      - ${PWD}:/src
```

Comment:
This mounts the all of your local files to `/src` in the image, overwriting any files that were copied in during the build. The docker build also copied everything to `/usr/src` not `/src`.

I think what you want to do is mount your local `/src` directory to the image, which would be something like `- ${PWD}/src:/usr/src/src`.